ui/styles.css

```diff
@@ -103,3 +103,7 @@ h5 {
 .markdown a.mention {
   @apply pointer-events-none cursor-default;
 }
+
+.text-xxs {
```

Comment:
This should be done by "extending" the font-size in tailwind.config.js

Reply:
done